tasklist/src/pages/listProject/listProject.html

```diff
@@ -15,6 +15,13 @@
 </ion-header>
 
 <ion-content>
+  <ion-item>
+    <ion-searchbar [(ngModel)]="searchTerm" (ionInput)="getDataFromApi()"></ion-searchbar>
+  </ion-item>
+  <ion-item>
+    <ion-label>Id</ion-label>
```

Comment:
The user don't know the ID and can't read it. You need to put the ID in the list and the detail of the project if you want filter by ID